src/daft-schema/src/schema.rs

```diff
@@ -18,8 +18,12 @@ use common_error::{DaftError, DaftResult};
 
 pub type SchemaRef = Arc<Schema>;
 
-#[derive(Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Display, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(transparent)]
+#[display("{}\n", make_schema_vertical_table(
```

Comment:
I think the former is actually more readable than this!

Reply:
Would you want the manual `impl Display for ...` to be used here? Or create a new wrapper helper function which would do something similar?